packages/charting/src/components/Legends/Legends.styles.ts

```diff
@@ -0,0 +1,49 @@
+import { ILegendsStyles, ILegendStyleProps } from './Legends.types';
```

Comment:
The colors in this file should also come from a theme.

Reply:
changed